static/src/javascripts/projects/common/modules/onward/history-containers.js

```diff
@@ -0,0 +1,127 @@
+/*
+ Module: history-containers.js
+ Description: Inject personalised containers based on reading history
+ */
+define([
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/ajax',
+    'Promise',
```

Comment:
third-party libs go first in the list - https://github.com/guardian/frontend/wiki/JavaScript-guidelines#defining-requirements